src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -23,16 +24,29 @@ export const ArtistHeader: SFC<Props> = props => {
   return (
     <Subscribe to={[AppState]}>
       {({ state }) => {
+        const {
+          mediator,
+          system: { currentUser },
+        } = state
+
         return (
           <Responsive>
             {({ xs }) => {
               if (xs) {
                 return (
-                  <SmallArtistHeader mediator={state.mediator} {...props} />
+                  <SmallArtistHeader
+                    mediator={mediator}
+                    currentUser={currentUser}
```

Comment:
Nice yea, something was clobbering the props so it'll be easier/clearer going forward to settle on grabbing it from the global state. 👍